src/components/RequireTwoFactorAuthenticationModal.tsx

```diff
@@ -47,6 +47,7 @@ function RequireTwoFactorAuthenticationModal({onCancel = () => {}, description,
                     type={shouldUseNarrowLayout ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED : CONST.MODAL.MODAL_TYPE.CONFIRM}
                     innerContainerStyle={{...styles.pb5, ...styles.pt3, ...styles.boxShadowNone}}
                     shouldEnableNewFocusManagement={shouldEnableNewFocusManagement}
+                    animationOutTiming={500}
```

Comment:
500 is new value, specific to this modal. If this looks much smoother than before, should this be applied to all modals for consistency?
I think we can leave as is for now so suggest to remove this line

Reply:
In general, we're now improving user experience related to modal in [this issue](https://github.com/Expensify/App/issues/49354). This is just a part of bigger change 😊
Here in this PR I was just fixing a small bug that I spotted, while analysing all modals in the app. But I thought it would better look with the longer slide out animation, that's why I asked the design team 📞
Soon all modals will be more consistent in case of animations.